### ISSUE TITLE
AI Alt Text: Update KBA to match latest UI

### ISSUE THREAD

**Issue by kellylein** [department-of-veterans-affairs/va.gov-cms#22580]
## User Story or Problem Statement
As a Drupal Editor, I want the KBA to match the way the UI appears within Drupal, so that I can more easily understand how the feature works.

## Description or Additional Context 
We need to go back into the AI Alt Text KBA and update the screenshot and text to match how the UI looks.  The button to regenerate the Alt Text is in a different spot and the text on the button is different.
Current KBA:
<img width="1013" height="825" alt="Image" src="https://github.com/user-attachments/assets/ba006d14-37fa-4907-9722-1783d912255d" />

Update to match Feature:
<img width="943" height="309" alt="Image" src="https://github.com/user-attachments/assets/dd52cd89-7fd8-4c0d-9caf-ba26433de8c7" />

## Acceptance Criteria
- [ ] Alt Text KBA descriptions matches feature exactly

**Comment by TroyCMSSupport** [comment]
@kellylein - I am unable to locate ANY button related to AI generation of Alt Text.
I have added new images, I have edited images, but none of the scenarios I tested had any sort of AI button. 
I double and triple checked both Training and Prod - No AI button at all.
I tried in Chrome and Edge browsers (I've seen cases where browser updates hide things like certain kinds of buttons).
I asked Edmund about it. He said the AI feature should be in place - even in production.
I kept thinking 'I must be doing something wrong' but I just don't see it.... 
I wonder if it was overwritten, or hidden by code, or some other situation... ?
So, with PTO upcoming, this will have to be rolled to the sprint after next....
